java-frontend/src/main/java/org/sonar/java/model/LiteralUtils.java

```diff
@@ -92,7 +92,11 @@ private static Integer minus(@Nullable Integer nullableInteger) {
   }
 
   public static boolean isEmptyString(Tree tree) {
-    return tree.is(Tree.Kind.STRING_LITERAL) && trimQuotes(((LiteralTree) tree).value()).isEmpty();
+    return tree.is(Kind.STRING_LITERAL) && trimQuotes(((LiteralTree) tree).value()).isEmpty();
+  }
+
+  public static boolean isNullOrWhitespace(Tree tree) {
+    return tree.is(Kind.NULL_LITERAL) || (tree.is(Kind.STRING_LITERAL) && StringUtils.isBlank(trimQuotes(((LiteralTree) tree).value())));
```

Comment:
I wouldn't put this method here. It seems quite specific to the rule we want to fix.